controllers/controller.js

```diff
@@ -28,9 +29,20 @@ const findSpecificArticle = (request, response, next) => {
     .catch(next);
 };
 
+const findArticleComments = (request, response, next) => {
+  if (isNaN(Number(request.params.article_id))) {
+    response.status(400).send({ msg: "400 - Bad request" });
+  }
+  findCommentsByArticleId(request.params.article_id)
+    .then((comment) => {
+      response.status(200).send({ comments: comment });
+    })
```

Comment:
you can shorten this down by declaring the parameter as `comments` instead - then you can just write `res.status(200).send({ comments });`